.tekton/scanner-v4-db-pull-request.yaml

```diff
@@ -0,0 +1,464 @@
+    - description: ""
+      name: CHAINS-GIT_URL
+      value: $(tasks.clone-repository.results.url)
+    - description: ""
+      name: CHAINS-GIT_COMMIT
+      value: $(tasks.clone-repository.results.commit)
+    - description: ""
+      name: JAVA_COMMUNITY_DEPENDENCIES
+      value: $(tasks.build-container.results.JAVA_COMMUNITY_DEPENDENCIES)
+
+    workspaces:
+    - name: workspace
+    - name: git-auth
+
+    tasks:
+
+    - name: init
+      params:
+      - name: image-url
+        value: $(params.output-image)
+      - name: rebuild
+        value: $(params.rebuild)
+      - name: skip-checks
+        value: $(params.skip-checks)
+      taskRef:
+        params:
+        - name: name
+          value: init
+        - name: bundle
+          value: quay.io/redhat-appstudio-tekton-catalog/task-init:0.2@sha256:3d8f01fa59596a998d30dc700fcf7377f09d60008337290eebaeaf604512ce2b
+        - name: kind
+          value: task
+        resolver: bundles
+
+    - name: clone-repository
+      params:
+      - name: url
+        value: $(params.git-url)
+      - name: revision
+        value: $(params.revision)
+      # A shallow repo clone is sufficient for scanner-v4-db build.
+      - name: depth
+        value: "1"
+      - name: fetchTags
+        value: "false"
+      runAfter:
+      - init
+      taskRef:
+        params:
+        - name: name
+          value: git-clone
+        - name: bundle
+          value: quay.io/redhat-appstudio-tekton-catalog/task-git-clone:0.1@sha256:982e53397367ea9680b5cc543f5cbfc8e90124ffb463551eea33e4477d0a7ec6
+        - name: kind
+          value: task
+        resolver: bundles
+      when:
+      - input: $(tasks.init.results.build)
+        operator: in
+        values:
+        - "true"
+      workspaces:
+      - name: output
+        workspace: workspace
+      - name: basic-auth
+        workspace: git-auth
+
+    - name: prefetch-dependencies
+      params:
+      - name: input
+        value: $(params.prefetch-input)
+      runAfter:
+      - clone-repository
+      taskRef:
+        params:
+        - name: name
+          value: prefetch-dependencies
+        - name: bundle
+          value: quay.io/redhat-appstudio-tekton-catalog/task-prefetch-dependencies:0.1@sha256:e29adab9f66415b3be2e89e154c03ec685900fdad90051a555d7d027f94f874e
+        - name: kind
+          value: task
+        resolver: bundles
+      workspaces:
+      - name: source
+        workspace: workspace
+
+    - name: fetch-db-init-bundle
+      runAfter:
+      - init
+      - clone-repository
+      taskSpec:
+        steps:
+          - name: fetch-db-init-bundle
+            image: registry.access.redhat.com/ubi8-minimal:latest
+            script: |
+              curl --silent --retry 3 --show-error --fail --output "$(workspaces.source.path)/source/db-init.dump.zst" \
+                "https://storage.googleapis.com/scanner-v4-test/database-init-bundles/db-init-dev.dump.zst"
```

Comment:
Is `scanner-v4-test` our GCloud bucket name? It does not "smell" like quite a production dependency. Let's check with the Scanner team if they plan more stable endpoint and how would it function for different ACS releases. There's quite elaborate setup for scanner-v4 blobs that was created here https://github.com/stackrox/stackrox/pull/10365 and I'm surprised the one for v4-db is so simple.

Reply:
Same https://redhat-internal.slack.com/archives/C033SKHG9CZ/p1710776736986199 applies.

Reply:
This is no longer relevant since this tekton task was removed. (7c591b8)